main.js

```diff
@@ -18,8 +18,10 @@
  *        MIT License. All code unless otherwise specified is
  *        Copyright (c) Matthew Knox and Contributors 2015.
  */
-
+'use strict';
 // Load NodeJS Modifications/Variables
+require('babel-register');
+require('babel-polyfill');
```

Comment:
prototypes.js is the second thing loaded after these at the moment, and is a singleton so will never be disposed of (hence will act like it is in this file). We don't want to encourage additions to the require.js so putting it in prototypes.js should have no side effects unless they have done something very weird in their code.